.github/workflows/build_test_deploy.yml

```diff
@@ -77,9 +78,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - run: yarn install --frozen-lockfile --check-files
+      - run: yarn install --immutable
+      - run: yarn prepublish
 
       - run: |
+          rm ./.yarnrc.yml
```

Comment:
Otherwise the PnP test accidentally inherits the top-level settings, including the `nodeLinker: node-modules` flag - which would be a bit counter-productive given the purpose of the test 😄

Reply:
How about creating and committing an empty `.yarnrc.yml` in a subfolder?

Reply:
The config is inherited from all folders above itself